op-program/chainconfig/chaincfg.go

```diff
@@ -2,123 +2,40 @@ package chainconfig
-	GrayGlacierBlock:              big.NewInt(4061224),
-	MergeNetsplitBlock:            big.NewInt(4061224),
-	BedrockBlock:                  big.NewInt(4061224),
-	RegolithTime:                  &params.OptimismGoerliRegolithTime,
-	TerminalTotalDifficulty:       big.NewInt(0),
-	TerminalTotalDifficultyPassed: true,
-	Optimism: &params.OptimismConfig{
-		EIP1559Elasticity:  10,
-		EIP1559Denominator: 50,
-	},
-}
-
-var OPSepoliaChainConfig = &params.ChainConfig{
-	ChainID:                       big.NewInt(11155420),
-	HomesteadBlock:                big.NewInt(0),
-	DAOForkBlock:                  nil,
-	DAOForkSupport:                false,
-	EIP150Block:                   big.NewInt(0),
-	EIP155Block:                   big.NewInt(0),
-	EIP158Block:                   big.NewInt(0),
-	ByzantiumBlock:                big.NewInt(0),
-	ConstantinopleBlock:           big.NewInt(0),
-	PetersburgBlock:               big.NewInt(0),
-	IstanbulBlock:                 big.NewInt(0),
-	MuirGlacierBlock:              big.NewInt(0),
-	BerlinBlock:                   big.NewInt(0),
-	LondonBlock:                   big.NewInt(0),
-	ArrowGlacierBlock:             big.NewInt(0),
-	GrayGlacierBlock:              big.NewInt(0),
-	MergeNetsplitBlock:            big.NewInt(0),
-	BedrockBlock:                  big.NewInt(0),
-	RegolithTime:                  &enabledFromBedrockBlock,
-	TerminalTotalDifficulty:       big.NewInt(0),
-	TerminalTotalDifficultyPassed: true,
-	Optimism: &params.OptimismConfig{
-		EIP1559Elasticity:  6,
-		EIP1559Denominator: 50,
-	},
-}
-
-var OPMainnetChainConfig = &params.ChainConfig{
-	ChainID:                       big.NewInt(10),
-	HomesteadBlock:                big.NewInt(0),
-	DAOForkBlock:                  nil,
-	DAOForkSupport:                false,
-	EIP150Block:                   big.NewInt(0),
-	EIP155Block:                   big.NewInt(0),
-	EIP158Block:                   big.NewInt(0),
-	ByzantiumBlock:                big.NewInt(0),
-	ConstantinopleBlock:           big.NewInt(0),
-	PetersburgBlock:               big.NewInt(0),
-	IstanbulBlock:                 big.NewInt(0),
-	MuirGlacierBlock:              big.NewInt(0),
-	BerlinBlock:                   big.NewInt(3950000),
-	LondonBlock:                   big.NewInt(105235063),
-	ArrowGlacierBlock:             big.NewInt(105235063),
-	GrayGlacierBlock:              big.NewInt(105235063),
-	MergeNetsplitBlock:            big.NewInt(105235063),
-	BedrockBlock:                  big.NewInt(105235063),
-	RegolithTime:                  &enabledFromBedrockBlock,
-	TerminalTotalDifficulty:       big.NewInt(0),
-	TerminalTotalDifficultyPassed: true,
-	Optimism: &params.OptimismConfig{
-		EIP1559Elasticity:  6,
-		EIP1559Denominator: 50,
-	},
+func init() {
+	mustLoadConfig := func(chainID uint64) *params.ChainConfig {
+		cfg, err := params.LoadOPStackChainConfig(chainID)
+		if err != nil {
+			panic(err)
+		}
+		return cfg
+	}
+	OPGoerliChainConfig = mustLoadConfig(420)
```

Comment:
can use use the constants for Goerli/Sepolia/Mainnet chain IDs here?

Reply:
The intention is to remove those global configs once the superchain-registry is stable/approved. So I don't want the globals in the critical path. The test will ensure that the registry behaves as expected for these important configs.